Makefile

```diff
@@ -3,6 +3,7 @@
 
 PROJECT_NAME             := upf-epc
 VERSION                  ?= $(shell cat ./VERSION)
+GO                  	 ?= go
```

Comment:
I believe it's useless to map a command to env variable. Use simply `go` directly in the Make target.